### ISSUE TITLE
Final updates to ROI content type

### ISSUE THREAD

**Issue by EWB** [EWB/dslc#7]
After checking on the recent updates, please make a final(ish) round of updates to the content type - as needed - on Dev.  This would include adding the date info - using a date field with year/month/day format.  Thanks

**Comment by cainaru** [comment]
- Added Date of Admission as a date Field Type with Text Field. In the field settings, I set the date attributes to collect as Year, Month, and Day. In the More Settings and Values for the field, I set the date entry options as Jan 20 2016 (seeing as how the paper forms typically have the dates as something like Dec 15 1910).

In the Manage Display for the Date of Admission, it has the date to be displayed to the user as using the Long (Wed, 01/20/2016 - 14:06) format. Should this be something different?

**Comment by EWB** [comment]
thanks, yes re: the date - should be something a bit more human-friendly in terms of the formatter.  If that is familiar territory please switch it - otherwise I am happy to show you how or can do it myself and then show you later....

**Comment by cainaru** [comment]
The way I was going to do it was go into admin/config/regional/date-time, click Add date type, name the Date type as Admission, and then select something like January 20, 2016 - 2:16pm from the dropdown menu for Date format. Is that the best way to do it?

**Comment by EWB** [comment]
Yes, perfect - sometimes this requires a custom date format - but this works, thanks

**Comment by cainaru** [comment]
Great, should be all set now!

**Comment by EWB** [comment]
excellent, thanks, I will re-roll the feature and add it to prod shortly

**Comment by EWB** [comment]
updates have been rolled into feature and pushed to production...
